extensions/stock-api-flask/manifest.json

```
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "locale": "en_US.UTF-8",
+  "metadata": {
+    "appmode": "python-api",
+    "entrypoint": "app"
+  },
+  "extension": {
+    "name": "stock-api-flask",
+    "title": "Flask Stock Pricing Service",
+    "description": "APIs are a great way for software engineering teams to use your models without translating them into different languages",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/stock-api-flask",
+    "version": "0.0.0"
```

Let's release it to add it to the gallery 🎉 

```suggestion
    "version": "1.0.0"
```